pypeit/spectrographs/keck_deimos.py

```diff
@@ -604,47 +662,140 @@ def get_rawimage(self, raw_file, det):
         detlsize = hdu[0].header['DETLSIZE']
         x0, x_npix, y0, y_npix = np.array(parse.load_sections(detlsize)).flatten()
 
-        # Create final image
-        if det is None:
-            image = np.zeros((x_npix, y_npix + 4 * postpix))
-            rawdatasec_img = np.zeros_like(image, dtype=int)
-            oscansec_img = np.zeros_like(image, dtype=int)
-
         # get the x and y binning factors...
         binning = hdu[0].header['BINNING']
         if binning != '1,1':
             msgs.error("This binning for DEIMOS might not work.  But it might..")
 
-        # DEIMOS detectors
-        nchip = 8
+        # get the chips to read in
+        # DP: I don't know if this needs to still exist. I believe det is never None
+        if det is None:
```

Comment:
I think this might have been used when trying to generate an image of all 8 detectors.  But can't quire recall.